README.md

```diff
@@ -1,8 +1,8 @@
 # KFServing
-KF Serving provides a Custom Resource Definition for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and Custom containers.
+KFServing provides a [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve majority of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. Using KFServing you should be able to deploy, scale up or down to zero and perform A/B tests on your models in a consistent and standardized way.
```

Comment:
I think 80% conveys the intended meaning better than "majority". It's a good framework for thinking following the 80-20 rule.

Reply:
Yeah - its just that it conveys that we have 100% of the use cases, covering 80% here. A number to me looked a bit out of place.

Reply:
Should we add a link to https://en.wikipedia.org/wiki/Pareto_principle

Reply:
Think it would distract from readability.  More layman language, the better for first page.

Reply:
Also if i see a hyperlink there, expectation would be it points me to all the model serving use cases detailed, and points what all we are covering